client/js/components/procedure/DpSimplifiedNewStatementForm.vue

```diff
@@ -139,7 +139,7 @@
                     :size="5" />
                   <dp-input
                     id="r_orga_city"
-                    v-model="values.submitter.city"
+                    v-model="values.submitter.ort"
```

Comment:
Is there a possibility that `data` can adjust this? I am not a huge fan of mixing english and german variable names in the code. Also I don't understand why this was changed to `ort`. 

Besides from that code lgtm

Reply:
Yes, I agree. I asked data if they can do it fast, otherwise I would change it here, because the sprint is already over.